realtime/server.js

```diff
@@ -51,18 +53,23 @@ winston.info('Ting real-time server v1 listening on port ' + config.node.port +
 winston.debug('Debug logging is enabled. Disable it if you see too many logs.');
 winston.debug('Using persistence API back-end at ' + URL);
 
+//when user is connected that function is executed
```

Comment:
This can be understood by reading the library's documentation. The purpose of the comments should not be to document a third party library.